MapboxGeocoder/MBPlacemark.swift

```diff
@@ -486,6 +491,13 @@ open class GeocodedPlacemark: Placemark {
         return properties?.maki
     }
     
+    /**
+     A numerical score from 0 (least relevant) to 0.99 (most relevant) measuring
+     how well each returned feature matches the query. You can use the this
+     property to remove results that don’t fully match the query.
+     */
+    open var relevance: Double?
```

Comment:
Make this property non-optional so that Objective-C code can access it. Use −1 to mark the absence of a `relevance` in the API response. According to [the documentation](https://www.mapbox.com/api-documentation/#response-object) and [this line in Carmen](https://github.com/mapbox/carmen/blob/3c04a5216fe2b87053c667765b08e20c44f50683/lib/geocoder/verifymatch.js#L384), we can count on a present value to never dip below 0.